samples/sqlservices/tasks/buildtasks.js

```diff
@@ -83,7 +83,7 @@ gulp.task('compile:test', function (done) {
 });
 
 // COMPOSED GULP TASKS /////////////////////////////////////////////////////
-gulp.task("compile", gulp.series("compile:src", "compile:test"));
+gulp.task("compile", gulp.series("compile:src"));
```

Comment:
we don't have any test code, as a result the compile test is only compiling the azdata.proposed.d.ts and it will fail.

Reply:
Just get rid of this and rename `compile:src` to just `compile`. No reason to have extra tasks like this if it's only going to call the other.